src/lib/plugins/blocks.js

```diff
@@ -4,12 +4,7 @@ module.exports.player=function(player,serv)
 {
   player.changeBlock=async (position,blockType,blockData) =>
   {
-    serv.players
-      .filter(p => p.world==player.world && player!=p)
```

Comment:
@rom1504 The vanilla server echos the block change back to the client, so I think we should too.